Candle led

moonflowerrrr opened the pull request:
auton LEDs have not been tested and the disabled LEDs do not work

moonflowerrrr committed:
Created CANdle leds
added multiple colors

moonflowerrrr committed:
Created an enum for the CANdle
- added multiple colors inside of it
- made a function that controls the LEDS
- edited the LEDControlCommand so that the LEDs were now CANdle

moonflowerrrr committed:
Added an enum that separates the LED strips from each other

moonflowerrrr committed:
Tuned colors for LEDs

dlozon commented on src/main/java/frc/robot/Constants.java:
```diff
@@ -123,6 +121,51 @@ public static enum ClimberPosition {
+
+        public static enum LEDStrip {
+            CANDLE(0, 8),
+            SHOOTER(8, 0),
```
We still have to finish filling out the ledstrip values

dlozon commented on src/main/java/frc/robot/Constants.java:
```diff
@@ -21,10 +21,8 @@ public static final class ControllerPorts {
-        public static final int SHOOTER_LEDS_PORT = 0;
-        public static final int INTAKE_LEDS_PORT = 1;
+    public static final class LEDPorts {
+        public static final int LED_PORT = 2;
```
should be named in a way that indicates that candle uses a CAN id and not a port on the rio

dlozon commented on src/main/java/frc/robot/subsystems/Candle.java:
```diff
@@ -0,0 +1,24 @@
+
+    public CANdle candle;
+
+    /** */
```
this class should be commented

moonflowerrrr replied:
okay g

moonflowerrrr committed:
Added comments in the Candle class
Changed the name for the ID of the Candle to be more accurate
Added the starting index and the strip length for Shooter and Intake

moonflowerrrr committed:
Added comments to the Intake class

moonflowerrrr committed:
Merge branch 'main' of https://github.com/TorqueNados-FRC5090/realbot-2024 into candle-led
